udm-le.sh

```diff
@@ -212,8 +212,8 @@ install_lego)
 	;;
 renew)
 	echo "renew(): Attempting certificate renewal"
-	echo "renew(): ${LEGO_BINARY} --path \"${LEGO_PATH}\" ${LEGO_ARGS} renew --days 60"
-	${LEGO_BINARY} --path "${LEGO_PATH}" ${LEGO_ARGS} renew --days 60 && deploy_certs && restart_services
+	echo "renew(): ${LEGO_BINARY} --path \"${LEGO_PATH}\" ${LEGO_ARGS} renew --days ${CERT_DAYS_BEFORE_RENEWAL}"
```

Comment:
I wonder if we should be setting a default here, e.g. `${CERT_DAYS_BEFORE_RENEWAL:-30}` so that if someone doesn't update their udm-le.env but does update the script side it won't fail?

Reply:
`CERT_DAYS_BEFORE_RENEWAL` is set to 30 in the `udm-le.env` because by default lego sets it at 30 - https://github.com/go-acme/lego/blob/fa7cf5cd4f487195d42634590e5d64c3b83c3bf2/docs/data/zz_cli_help.toml#L86

Reply:
What I'm saying is, a lot of people just check out the latest copy of the script and copy it over, and don't diff out any changes in the udm-le.env file. So, the more changes you make in the script that are dependent on variables in the .env without having a default in the script, the more support issues I get because people don't look at the release notes or the diffs between releases. This way, even though it seems redundant, if people don't read the release notes it'll continue working vs exiting because lego gets `--days` with no value.